maintainers/scripts/pluginupdate.py

```diff
@@ -195,21 +203,46 @@ def __init__(
         name: str,
         root: Path,
         get_plugins: str,
+        # TODO modifier pour accepter un plugin
```

Comment:
TODO: traduire

Reply:
I removed the comment. I think generate_nix should be a member, as is done here. I've kept the argument to minimize changes.